snap/snapcraft.yaml

```diff
@@ -239,6 +252,7 @@ parts:
       - openscad  # OpenSCAD
       - povray # Raytracing
       - povray-includes # Raytracing
+      - openscad
```

Comment:
although, openscad is already included  3 lines up. So is this line necessary ?

Reply:
It is not! I am surprised snapcraft didn't fail. New PR to remove.
https://github.com/FreeCAD/FreeCAD-snap/pull/94

Reply:
Thanks: we can drop those povray installs as well, Raytracing is gone.